app/models/user.rb

```diff
@@ -7,6 +7,7 @@ class User
   # Use the User database
   store_in client: "users", collection: "back_office_users"
 
+  field :active, type: Boolean, default: true
```

Comment:
Since this is a new field, in order for all this to work we will need to either change the `active?` method to return `true` in case of `nil` or we can add a `deactivated` field to check instead, as Devise does

Reply:
Good spot, assumed that `default: true` would handle this but turns out it doesn't! Fixed.